app/controllers/index.js

```diff
@@ -0,0 +1,32 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import ENV from 'website-my/config/environment';
+
+const BASE_URL = ENV.BASE_API_URL;
+
+export default class IndexController extends Controller {
+  @tracked status = this.model;
+
+  @action async updateStatus(status) {
+    this.status = status;
+    try {
+      const response = await fetch(`${BASE_URL}/users/self`, {
+        method: 'PATCH',
+        body: JSON.stringify({
+          status,
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'include',
+      });
+      const { status } = response;
+      if (status !== 204) {
+        alert('Something went wrong. Please check console errors.');
+      }
+    } catch (error) {
+      console.error('Error : ', error);
```

Comment:
The user doesn't SEE anything on the UI? A normal non-technical user doesn't know/care about the console

Reply:
So should we display some error page in the UI ?? Currently user will not see anything.

Reply:
Yes, a simple alert should do the job for now

Reply:
nothing here also?

Reply:
I have changed the alert message to `Something went wrong!`